fe/fe-core/src/main/java/com/starrocks/statistic/AnalyzeManager.java

```diff
@@ -35,11 +35,11 @@
 
     private final Map<Long, AnalyzeJob> analyzeJobMap;
 
-    private ExecutorService executor;
+    private final static ExecutorService executor =
+            ThreadPoolManager.newDaemonFixedThreadPool(1, 16, "analyze-replay-pool", true);
 
     public AnalyzeManager() {
         analyzeJobMap = Maps.newConcurrentMap();
-        executor = ThreadPoolManager.newDaemonFixedThreadPool(1, 16, "analyze-replay-pool", true);
```

Comment:
Why not AnalyzeManager is single  instance?

Reply:
The other Managers in the Catalog are not single instances either.
Catalog calls getCurrentAnalyzeMgr(), which I'm worried will introduce other problems when doing checkpoint.
```
Catalog.getCurrentAnalyzeMgr().write(dos);
```